mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearAlgorithm.scala

```diff
@@ -79,9 +79,6 @@ abstract class StreamingLinearAlgorithm[
    * @param data DStream containing labeled data
    */
   def trainOn(data: DStream[LabeledPoint]): Unit = {
-    if (model.isEmpty) {
```

Comment:
Hmm. It says here that, the initial weights are set to zero. https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearRegressionWithSGD.scala#L82 . And it seems that that was the intent previously as well and it was incorrectly set as `Vectors.dense(numFeatures)`

Should this (https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearRegressionWithSGD.scala#L62) be changed to a non-empty model?